vertical-pod-autoscaler/enhancements/7862-cpu-startup-boost/README.md

```diff
@@ -95,24 +95,37 @@ down the CPU resources to the appropriate non-boosted value:
 
 ### API Changes
 
-The new `StartupBoost` parameter will be added to the [`ContainerResourcePolicy`](https://github.com/kubernetes/autoscaler/blob/vertical-pod-autoscaler-1.3.0/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L191)
-and contain the following fields:
-  * `StartupBoost.CPU.Factor`: the factor by which to multiply the initial
-  resource request and limit of the containers' targeted by the VPA object.
-  * `StartupBoost.CPU.Value`: the target value of the CPU request or limit
-  during the startup boost phase.
-  * [Optional] `StartupBoost.CPU.Duration`: if specified, it indicates for how
-  long to keep the pod boosted **after** it goes to `Ready`.
+The new `StartupBoost` parameter will be added to both:
```

Comment:
Is adding the option to both the spec and the container resource policy simply just for convenience?

Reply:
Makes sense to me, and looks good to me. :)